src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-logs-page.mdx

```diff
@@ -9,9 +9,9 @@ metaDescription: Here are some tips for understanding the OpenTelemetry logs pag
 
 After you've sent us your OpenTelemetry data and opened your service (entity) in the UI, you can click on **Logs** to see data about your application. For more information about how to associate log data to your application in New Relic, see our [OpenTelemetry and logging documentation](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs).
 
-For your data to appear in this section, make sure it has the following:
+For your data to appear in this section, do the following:
 
-* `service.name`
-* To correlate with trace data, the logs should contain the `trace.id` and `span.id` attributes.
+* Make sure your logs are correlated with your service via `service.name`. For details, see [Correlate your application logs](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs#log-correlation).
+* To correlate with trace data, the logs need to include the trace context that is contained `trace_id` and `span_id`.  While OpenTelemetry instrumentation is likely to inject trace context as `trace_id` and `span_id`, the New Relic UI expects the fields to be `trace.id` and `span.id`. You'll need to configure rules in your log pipeline to translate `trace_id` to `trace.id` and `span_id` to `span.id`.
```

Comment:
This bullet feels a little unnecessarily wordy. I think you can get the point across with far fewer words :)

Reply:
@paperclypse I think we need the verbosity to ensure readers don't miss this subtle requirement, but I did change the final phrase to "...to translate `trace_id` and `span_id` to `trace.id` and `span.id`.